src/aria2p/api.py

```diff
@@ -850,3 +852,52 @@ def stop_listening(self) -> None:
         if self.listener:
             self.listener.join()
         self.listener = None
+
+    def input_downloads(self, lines):
+        """
+        Helper to split downloads in an input file.
+
+        Arguments:
+             lines:  URI string.
+        """
+        block = []
+        for line in lines:
+            if line.lstrip().startswith("#"):  # Ignore Comments
+                continue
+            if not line.strip():  # Ignore empty line
+                continue
+            if not line.startswith(" "):  # Read download option
```

Comment:
The comment is not right, it's the opposite: when we enter this condition it means we are reading a line containing URIs.